Sources/MuxPlayerSwift/PublicAPI/Options/MonitoringOptions.swift

```diff
@@ -9,10 +9,11 @@ import MuxCore
 /// Options to customize monitoring data reported by Mux
 public struct MonitoringOptions {
 
-    /// Environment key associated with the monitoring data
+    // Environment key associated with the monitoring data
```

Comment:
question: Is this desired?

Reply:
Good question, yes, `environmentKey` is not public so it shouldn't appear in the docc documentation catalog / generated static pages.